app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -98,24 +127,182 @@ def test_serializer_outputs_expected_schema(self):
         }
 
         self.assertEqual(
-            set(serialized.data.keys()), set(expected_data.keys())
+            set(serializer.data.keys()), set(expected_data.keys())
         )
-        self.assertEqual(serialized.data, expected_data)
+        self.assertEqual(serializer.data, expected_data)
 
     def test_serializer_locales(self):
         locale = LocaleFactory()
         experiment = ExperimentFactory.create(locales=[locale])
-        serialized = ExperimentSerializer(experiment)
+        serializer = ExperimentSerializer(experiment)
         self.assertEqual(
-            serialized.data["locales"],
+            serializer.data["locales"],
             [{"code": locale.code, "name": locale.name}],
         )
 
     def test_serializer_countries(self):
         country = CountryFactory()
         experiment = ExperimentFactory.create(countries=[country])
-        serialized = ExperimentSerializer(experiment)
+        serializer = ExperimentSerializer(experiment)
         self.assertEqual(
-            serialized.data["countries"],
+            serializer.data["countries"],
             [{"code": country.code, "name": country.name}],
         )
+
+
+class TestFilterObjectBucketSampleSerializer(TestCase):
+
+    def test_serializer_outputs_expected_schema(self):
+        experiment = ExperimentFactory.create(
+            population_percent=Decimal("12.34")
+        )
+        serializer = FilterObjectBucketSampleSerializer(experiment)
+        self.assertEqual(
+            serializer.data,
+            {
+                "type": "bucketSample",
+                "input": ["normandy.recipe.id", "normandy.userId"],
+                "start": 0,
+                "count": 123,
```

Comment:
I don't think I agree with this. I think that we should do one of the following:

* Increase the sample to be per-10,000 instead of permilli (by making count 10,00), and use 1,234 as the count
* Use 123.4 as the count.
* Not allow less than 1-in-1000 (0.1%) precision in experiments.

We recently had an experiment that wanted a .67% sample, which is fine. We did a 67 out of 10,000 sample. As implemented, we probably would have lost that detail and run a differently sized sample. It's not that bad, but we should probably be faithful in preserving those details.

Reply:
A long time ago I asked ilana what the smallest meaningful population size should be and and she told me 0.01% so Experimenter stores the population percent as a Decimal that allows precision to that depth, so yes we should do 10 000 as you say.